Dockerfile-quickstart

```diff
@@ -0,0 +1,18 @@
+FROM sugarush/cayley
```

Comment:
I planned on changing it to Quay, though personally I avoid Quay and deploy my own registries.